web/src/app/pages/edit-survey/edit-survey.component.spec.ts

```diff
@@ -65,7 +89,13 @@ describe('EditSurveyComponent', () => {
     activeSurvey$ = new Subject<Survey>();
     surveyServiceSpy.getActiveSurvey$.and.returnValue(activeSurvey$);
 
+    const routes = [
```

Comment:
Are these routes being used for anything in these tests? Do they populate the router links or something?

Reply:
It was used by the `RouterTestingModule`, I thought I need that for the test environment, but it's not actually necessary, I removed it.